randovania/games/super_metroid/json_data/Brinstar.txt

```diff
@@ -817,13 +839,25 @@ Extra - asset_id: 96
   * Layers: default
   * Normal Door to Blue Brinstar Energy Tank Room/Door to Blue Brinstar Boulder Room
   > Door to Blue Brinstar Double Missile Room
-      Gravity Suit
+      All of the following:
+          Any of the following:
+              # Get out of the water
+              Gravity Suit or Movement (Intermediate)
+          Any of the following:
+              # Get over boulder pit
+              X-Ray Scope or Knowledge (Beginner)
 
 > Door to Blue Brinstar Double Missile Room; Heals? False
   * Layers: default
   * Normal Door to Blue Brinstar Double Missile Room/Door to Blue Brinstar Boulder Room
   > Door to Blue Brinstar Energy Tank Room
-      Trivial
+      All of the following:
+          Any of the following:
+              # Get out of the water
+              Gravity Suit or Movement (Intermediate)
+          Any of the following:
+              # Get over boulder pit
+              X-Ray Scope or Knowledge (Beginner)
```

Comment:
Marking this as trivial was probably correct. Getting out of the water at the far end of the room and back into the door is not really difficult.

Reply:
From experience of trying that, i disagree. But would be fine with lowering it down to a beginner movement

Reply:
Fine